Mac_BSD_resources.py

```diff
@@ -61,7 +61,7 @@ def measure_resources():
 
   # Next, get physical memory (RAM)
   phys_mem = "unable to find size of physical memory"
-  pipe = getShellPipe("sysctl hw.physmem")
+  pipe = getShellPipe("sysctl hw.memsize")
   for line in pipe:
     line_s = line.split(" ")
```

Comment:
You may even provide an example (hw.memsize: 8589934592), or use the expected `hw.memsize` at the start to parse the needed value.